src/ui/digital-exchange/components/SearchTextInput.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon } from 'patternfly-react';
+
+const SearchTextInput = ({
+  input,
+  filled,
+  meta,
+  onClear,
+  ...others
+}) => (
+  <div className="SearchBar__textbox">
+    <input
+      {...input}
+      id={input.name}
+      type="text"
+      className="SearchBar__textbox--base"
+      placeholder="Search"
+      {...others}
+    />
+    {filled ? (
+      <Button
+        className="btn-transparent SearchBar__button-close"
+        onClick={() => {
```

Comment:
please avoid passing arrow function in render: see https://reactjs.org/docs/faq-functions.html for further details